include/Surelog/Common/PathId.h

```diff
@@ -0,0 +1,140 @@
+#ifndef SURELOG_PATHID_H
+#define SURELOG_PATHID_H
+#pragma once
```

Comment:
If you already have the include-guard, you don't need `#pragma once`. In fact, I'd probably leave it out as it is non-standard, so you can't rely on it working anyway. Include guards are the only one guaranteed to work.

Reply:
ah ok, yeah, let's remove it everywhere in a separate change.